src/plugins/iam/cli/create-role.js

```diff
@@ -1,55 +1,47 @@
 'use strict';
 
 const path = require('path');
+
+// Nice ES6 syntax
+// const { Promise, _, icli } = require('@lager/lager/lib/lager').import;
 const lager = require('@lager/lager/lib/lager');
-const Promise = lager.getPromise();
+const Promise = lager.import.Promise;
+const _ = lager.import._;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["_" is defined but never used](https://www.codacy.com/app/lager/lager/file/2780163943/issues/source?bid=3360491&fileBranchId=3365294#l9)